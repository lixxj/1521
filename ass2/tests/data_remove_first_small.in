a = malloc 10
b = malloc 100
free a