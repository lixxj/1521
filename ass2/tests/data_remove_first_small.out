+00000 (F, 5000) 
[a] +00008 
+00000 (A,   20) +00020 (F, 4980) 
[a] +00008 [b] +00028 
+00000 (A,   20) +00020 (A,  108) +00128 (F, 4872) 
[b] +00028 
+00000 (F,   20) +00020 (A,  108) +00128 (F, 4872) 
+00000 (F,   20) +00020 (A,  108) +00128 (F, 4872) 
